zmq-sys/Cargo.lock

```diff
@@ -0,0 +1,44 @@
+[[package]]
```

Comment:
Shouldn't this file be gone?  I thought we removed zmq?

Reply:
@zippy that would be on 0.0.6